protocols/grpc/hyperstart.proto

```diff
@@ -110,20 +113,46 @@ message CreateSandboxRequest {
 message DestroySandboxRequest {
 }
 
+enum IpFamily {
+	v4 = 0;
+	v6 = 1;
+}
+
 message IpAddress {
-	string address = 1;
-	string mask = 2;
+	IpFamily family = 1;
+	string address = 2;
+	string mask = 3;
+}
+
+message Interface {
+	string device = 1;
+	string name = 2;
+	repeated IpAddress ipAddresses = 3;
+	uint64 mtu = 4;
+	string hwAddr = 5;
+}
+
+enum UpdateType {
+	AddIP = 0;
+	DelIP = 1;
+	SetName = 2;
+	SetMtu = 3;
+}
+
+message UpdateInterface {
+	UpdateType type = 1;
+	Interface interface = 2;
```

Comment:
Do we need the whole Interface  struct as identifier? 
Maybe a name could be enough？

Reply:
@WeiZhang555 You need the whole interface to pass the information you want to see updated.
So e.g. if type is `SetMTU`, we will get the MTU from the `Interface` and discard all other fields.

Reply:
Oh, yes, you’re right. I made a big mistake :smile: